src/app/ipv/router.js

```diff
@@ -38,10 +39,11 @@ router.get(
   renderAttemptRecoveryPage,
 );
 
-router.get(
-  getPagePath(PAGE_IPV_IDENTITY_DOCUMENT_TYPES),
-  staticPageMiddleware(PAGE_IPV_IDENTITY_DOCUMENT_TYPES),
-);
+const pages = [PAGE_IPV_IDENTITY_DOCUMENT_TYPES, PYI_POST_OFFICE];
+
+pages.forEach((page) => {
+  router.get(getPagePath(page), staticPageMiddleware(page));
+});
```

Comment:
What's going on here?

Reply:
Removed it, I added it to try something and forgot to remove it :D thanks :)